### ISSUE TITLE
igraph.plot() does not work with `options(warn=2)`

### ISSUE THREAD

**Issue by ntamas** [igraph/rigraph#636]
**Describe the bug**

Turning all warnings into errors with `options(warn=2)` makes `plot()` fail even with a simple graph.

**To reproduce**

```
> options(warn=2)
> library(igraph)
> plot(make_graph("zachary"))
Error in plot.default(0, 0, type = "n", xlab = xlab, ylab = ylab, xlim = xlim,  :
  (converted from warning) partial argument match of 'frame' to 'frame.plot'
```

**Version information**

 - R/igraph version: latest `master` branch
 - R version: 4.2.1
 - Operating system: macOS

This is probably a historical artifact; I think that some time back in the past the `frame=...` argument of R's built-in `plot.default()` was renamed to `frame.plot`. Our `plot.igraph` function forwards certain arguments to `plot()` but did not follow up the renaming of `frame` to `frame.plot`. This should be an easy fix if my understanding is correct, but I'm not sure whether I've gotten it right. cc @krlmlr

**Comment by ntamas** [comment]
Fixing this is needed to let me track down _another_ warning that appears while building the vignette. I cannot get a traceback on that _other_ warning because to do so I need to convert _all_ warnings into errors, but then I get caught up in the warning reported in this issue.

**Comment by ntamas** [comment]
Fixed in b252eac9
